rnd/autogpt_server/autogpt_server/executor/manager.py

```diff
@@ -65,32 +69,46 @@ def wait(f: Coroutine[T, Any, T]) -> T:
     # Execute the node
     prefix = get_log_prefix(graph_exec_id, node_exec_id, node_block.name)
     logger.warning(f"{prefix} execute with input:\n`{exec_data}`")
+
     wait(execution_update(node_exec_id, ExecutionStatus.RUNNING))
 
+    # TODO: Remove need for multiple database lookups
+    execution_result = get_execution_result(graph_exec_id, node_exec_id)
+    agent_server.send_execution_update(execution_result)  # type: ignore
```

Comment:
These can be bundled a long with `execution_update` call into a helper function ?
I guess we can simply send_execution_update by sending the node_exec_id and its status, so no need for `get_execution_result`. The one that will do it once will just be the agent_server.